x/mint/README.md

```diff
@@ -0,0 +1,128 @@
+# `x/mint`
+
+Adapted from <https://github.com/celestiaorg/celestia-app/tree/main/x/mint>
+TODO: fix the doc
+
+---
+
+## Abstract
+
+celestia-app's `x/mint` is a fork of the Cosmos SDK [`x/mint`](https://github.com/cosmos/cosmos-sdk/tree/5cd0b2316a7103468af38eab5d886f9f069c9cd7/x/mint) module that makes some changes to the inflation mechanism. The changes were motivated by a desire for Celestia to have a pre-determined inflation schedule. See [ADR-019](../../docs/architecture/adr-019-strict-inflation-schedule.md) for more details.
```

Comment:
probably all references to celestia would need to be deleted here

Reply:
Yep this will be done after we have the tokenomics and make this PR ready for review